sensor_msgs/Image[] images
float32[] rectangles    # her görüntü için [x, y, w, h] olarak düz bir dizi
string[] class_labels   # her görüntü için karşılık gelen sınıf etiketi ("" olabilir)